package.json

```diff
@@ -25,27 +25,34 @@
     "url": "https://github.com/jupyterlab/jupyterlab-metadata-service.git"
   },
   "scripts": {
-    "build": "tsc",
-    "watch": "tsc -w",
-    "lint": "stylelint style.css",
-    "clean": "rimraf lib tsconfig.tsbuildinfo",
-    "prepare": "npm run clean && jlpm run lint && npm run build"
+    "all": "jlpm run clean && jlpm run build && jlpm run build:jupyter",
+    "all:watch": "jlpm run clean && jlpm run build && jlpm run build:jupyter:watch",
+    "build": "jlpm run build:dev",
+    "build:dev": "jlpm install && jlpm run clean:packages && jlpm run build:packages && jlpm run link:packages && jlpm run install:extensions && jupyter labextension list",
+    "build:jupyter": "jupyter lab build && jupyter lab",
+    "build:jupyter:watch": "jlpm run build:jupyter && jupyter lab --watch",
```

Comment:
Not sure if this one makes sense, since you are running `jupyter lab` then `jupyter lab --watch`. 

We usually either wanna run `tsc && jupyter lab build && jupyter lab` or run `tsc --watch` and in another process `jupyter lab --watch`

Reply:
You're right. Fixed.